Controllers/AccountController.cs

```diff
@@ -24,17 +27,23 @@ public async Task LogOut()
             .WithRedirectUri(Url.Action("Index", "Index")!)
             .Build();
 
-        await HttpContext.SignOutAsync(Auth0Constants.AuthenticationScheme, authenticationProperties);
+        await HttpContext.SignOutAsync(
+            Auth0Constants.AuthenticationScheme,
+            authenticationProperties
+        );
         await HttpContext.SignOutAsync(CookieAuthenticationDefaults.AuthenticationScheme);
     }
 
     public async Task SignUp()
     {
         var authenticationProperties = new LoginAuthenticationPropertiesBuilder()
             .WithParameter("screen_hint", "signup")
-            .WithRedirectUri(Url.Action("CreateProfile", "Index")!)
+            .WithRedirectUri(Url.Action("Profile", "Index")!)
             .Build();
 
-        await HttpContext.ChallengeAsync(Auth0Constants.AuthenticationScheme, authenticationProperties);
+        await HttpContext.ChallengeAsync(
+            Auth0Constants.AuthenticationScheme,
+            authenticationProperties
+        );
```

Comment:
were you able to adjust the code styler so that it wouldn't indent lines like this?

Reply:
I'm not opposed to this